README.md

```diff
@@ -45,48 +46,40 @@ Releases can be found on the [release list][releases].
 
 
 For semi-official builds and third-party packages please see
-[mpv.io](http://mpv.io/installation/).
+[mpv.io/installation](http://mpv.io/installation/).
 
-## Changelog
+## Changelogs
```

Comment:
"A changelog is a log or record of all notable changes made to a project."
That's exactly what the mpv changelog is. A record of all notable changes.
mpv has one single changelog. One single record of all changes.
It doesn't have many changelogs., does it?
It's singular.